build/commands/scripts/commands.js

```diff
@@ -124,7 +125,8 @@ program
   .option('--notarize', 'notarize the macOS app with Apple')
   .option('--target_os <target_os>', 'target OS')
   .option('--target_arch <target_arch>', 'target architecture')
-  .option('--target_apk_base <target_apk_base>', 'target Android OS apk (classic, modern, mono)', 'classic')
+  .option('--target_apk_aab_base <target_apk_base>', 'target Android OS level for apk or aab (classic, modern, mono)', 'classic')
+  .option('--target_output <target_output>', 'target Android output format (apk, aab)', 'aab')
```

Comment:
if we don't plan to reuse this option for non-Android then it should be `target_android_output_format`

Reply:
Updated.